CommonTest.py

```diff
@@ -0,0 +1,68 @@
+import openpyxl
+import pandas as pd
+import glob
+
+class CommonTest:
+    def __init__(self):
+        self.total_rows = []
+        self.total_cols = []
+        self.workbook = None
+        self.results_ws = None
+        self.active_ws = None
+
+    def addCsvFiles(self, csv_files, output_file):
+        with pd.ExcelWriter(output_file, engine="xlsxwriter") as writer:
+            for index, csv_file in enumerate(csv_files):
+                sheet_name = f"AnalyzedData-{index}"
+                df = pd.read_csv(csv_file)
+                df.to_excel(writer, sheet_name=sheet_name, index=False)
+    
+                self.workbook = writer.book
+                self.active_ws = writer.sheets[sheet_name]
+
+                header_format = self.workbook.add_format({'bold': True})
+        
+                for col_num, value in enumerate(df.columns.values):
+                    self.active_ws.writer(0, col_num, value, header_format)
+                
+                for i, col_num  in enumerate(df.columns):
+                    max_length = max(df[col_num].astype(str).map(len).map(), len(col_num))
+                    self.active_ws.set_column(i, i, max_length + 2)
+
+                self.active_ws.freeze_panes(1,0)
+
+                rows, cols = df.shape
+                
+                self.total_rows.append(rows)
+                self.total_rows.append(cols)
+        
+        self._createResultsFile()
+        self._openResultsWorkbook(output_file)
+        self.workbook.close()
```

Comment:
Order of function calls is wrong.
```suggestion
        self.workbook.close()
        self._openResultsWorkbook(output_file)
```